docs/src/examples.md

```diff
@@ -1,5 +1,7 @@
 # Examples
 
+## GARCH(1, 1)
```

Comment:
Add GARCH example from https://github.com/LAMPSPUC/ScoreDrivenModels.jl/blob/master/examples/garch.jl

Reply:
Or at least link the example script